src/cli/cmd/login/mod.rs

```diff
@@ -40,8 +48,66 @@ impl CommandExecute for LoginSubcommand {
     }
 }
 
+pub async fn dnee_uds() -> color_eyre::Result<SendRequest<axum::body::Body>> {
+    let dnee_uds_socket_dir = Path::new("/nix/var/determinate");
+    let dnee_uds_socket_path = dnee_uds_socket_dir.join("determinatenixd.socket");
+
+    let stream = TokioIo::new(UnixStream::connect(dnee_uds_socket_path).await.unwrap());
+    let (mut sender, conn): (SendRequest<Body>, _) =
+        hyper::client::conn::http1::handshake(stream).await.unwrap();
+
+    // NOTE(colemickens): for now we just drop the joinhandle and let it keep running
+    let _join_handle = tokio::task::spawn(async move {
+        if let Err(err) = conn.await {
+            println!("Connection failed: {:?}", err);
+        }
+    });
+
+    let request = http::Request::builder()
+        .method(Method::GET)
+        .uri("http://dnee.unix/info")
```

Comment:
...which prevents the differences in these two places (`dnee.unix` in one place, `dnee.socket` in the other)

```suggestion
        .uri("http://localhost/info")
```